projects/fusion-grid/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "angular-fusiongrid",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Minor releases usually indicate that there are no breaking changes and it is backwards compatible, meaning existing functionality remains intact, and users can upgrade without significant modifications. But if we have a breaking change including changes that may cause existing functionality to no longer work as expected for users who are upgrading, then we should mark it as a major release. Do we have a confirmation on these data points?